psi4/src/read_options.cc

```diff
@@ -1052,12 +1052,15 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         /*- Enables the hybrid xc kernel in dispersion? !expert -*/
         options.add_bool("SAPT_DFT_DO_HYBRID", true);
         /*- Scheme for approximating exchange-dispersion for SAPT-DFT.
+        Previous to Nov 2022, default was ``FIXED`` with Hesselmann value.
         ``NONE`` Use unscaled ``Exch-Disp2,u`` .
         ``FIXED`` Use a fixed factor |sapt__sapt_dft_exch_disp_fixed_scale| to scale ``Exch-Disp2,u`` .
         ``DISP`` Use the ratio of ``Disp2,r`` and ``Disp2,u`` to scale ``Exch-Disp2,u`` . -*/
-        options.add_str("SAPT_DFT_EXCH_DISP_SCALE_SCHEME", "DISP", "NONE FIXED DISP");
-        /*- Exch-disp scaling factor for FIXED scheme for |sapt__sapt_dft_exch_disp_scale_scheme|. Default value of 0.686 suggested by Hesselmann and Korona, J. Chem. Phys. 141, 094107 (2014). !expert -*/
-        options.add_double("SAPT_DFT_EXCH_DISP_FIXED_SCALE", 0.686);
+        options.add_str("SAPT_DFT_EXCH_DISP_SCALE_SCHEME", "FIXED", "NONE FIXED DISP");
+        /*- Exch-disp scaling factor for FIXED scheme for |sapt__sapt_dft_exch_disp_scale_scheme|.
+        Default value of 0.770 suggested in Y. Xie, D. G. A. Smith and C. D. Sherrill, 2022 (submitted).
+        Previous to Nov 2022, default value was 0.686 suggested by Hesselmann and Korona, J. Chem. Phys. 141, 094107 (2014). !expert -*/
+        options.add_double("SAPT_DFT_EXCH_DISP_FIXED_SCALE", 0.769848);
```

Comment:
Okay. Probably this is the way to go because any accuracies after 3rd decimal place is meaningless because of the approximate nature of the scaling scheme. I think the reason I chose to put 3 decimal places in the paper is to follow what Hesselmann and Korona did in [Hesselmann and Korona, J. Chem. Phys. 141, 094107 (2014).].